### PR TITLE
docs: add doc french

### PR DESCRIPTION
**Pull request by Strawbang**
Hello, i'm French native, if you see any problem i'm open.

**Readme translation to French.**

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+Vous devriez maintenant être capable d'importer `"foo"` dans votre code et il sera dirigé vers la nouvelle définition de type.
+Ensuite, compilez *et* exécutez le code pour vous assurer que votre définition de type correspond bien à ce qui se passe à l'exécution.
+
+Une fois que vous avez testé vos définitions avec du code réel, faites un [PR](#faire-une-demande-de-pull-request)
+puis suivez les instructions pour [modifier un paquet existant](#modifier-un-paquet-existant) ou
+[créer un nouveau paquet](#créer-un-nouveau-paquet).
+
+### Faire une demande de pull request
+
+Once you've tested your package, you can share it on Definitely Typed.
```
Comment (sandersn) [inline README.fr.md:128]: This sentence is still English.

File: README.fr.md [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+<img src="https://github.com/DefinitelyTyped/dt-mergebot/blob/master/docs/dt-mergebot-lifecycle.svg">
+
+#### Partial clone
+
+<details>
+<summary>Vous pouvez cloner l'ensemble du dépôt <a href='https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository'>comme d'habitude</a>, mais il est volumineux et comprend un énorme répertoire de paquets de type.</summary>
+
+Vous pouvez cloner l'ensemble du dépôt [comme d'habitude](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository), mais il est volumineux et comprend un énorme répertoire de paquets de type. Cela prendra du temps à cloner et peut s'avérer inutilement lourd.
+
+Pour un clone plus facile à gérer qui inclut _only_ les paquets de type qui vous concernent, vous pouvez utiliser les fonctionnalités de git [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout), [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt), et [`--depth`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Cela réduira le temps de clonage et améliorera les performances de git.
```
Comment (sandersn) [inline README.fr.md:143]: Not sure this is the right word, but _only_ was still in English.

```suggestion
Pour un clone plus facile à gérer qui inclut _seulement_ les paquets de type qui vous concernent, vous pouvez utiliser les fonctionnalités de git [`sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout), [`--filter`](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---filterltfilter-specgt), et [`--depth`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Cela réduira le temps de clonage et améliorera les performances de git.
```

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+f("one");
+```
+
+Pour plus de détails, voir le readme de [dtslint](https://github.com/Microsoft/DefinitelyTyped-tools/tree/master/packages/dtslint#write-tests).
+
+#### Linter: `tslint.json`
+
+Le fichier de configuration du linter, `tslint.json` doit contenir `{ "extends" : "@definitelytyped/dtslint/dt.json" }`, et aucune règle supplémentaire.
+
+Si, pour une raison quelconque, une règle doit être désactivée, [désactivez-la pour cette ligne spécifique](https://palantir.github.io/tslint/usage/rule-flags/#comment-flags-in-source-code:~:text=%2F%2F%20tslint%3Adisable%2Dnext%2Dline%3Arule1%20rule2%20rule3...%20%2D%20Désactive%20les%20règles%20listées%20pour%20la%20prochaine%20ligne) en utilisant `// tslint:disable-next-line :[nomRègle]` - pas pour tout le paquet, afin que la désactivation puisse être examinée. (Il y a quelques anciennes configurations de lint qui ont des contenus additionnels, mais cela ne devrait pas se produire dans un nouveau travail).
```
Comment (sandersn) [inline README.fr.md:288]:
```suggestion
Si, pour une raison quelconque, une règle doit être désactivée, [désactivez-la pour cette ligne spécifique](https://palantir.github.io/tslint/usage/rule-flags/#comment-flags-in-source-code:~:text=%2F%2F%20tslint%3Adisable%2Dnext%2Dline%3Arule1%20rule2%20rule3...%20%2D%20Désactive%20les%20règles%20listées%20pour%20la%20prochaine%20ligne) en utilisant `// tslint:disable-next-line:[nomRègle]` - pas pour tout le paquet, afin que la désactivation puisse être examinée. (Il y a quelques anciennes configurations de lint qui ont des contenus additionnels, mais cela ne devrait pas se produire dans un nouveau travail).
```

File: README.fr.md [hunk 4]
```diff
@@ -0,0 +1,655 @@
...
+```json
+{
+    "private": true,
+    "type": "module"
+}
+```
+
+Ceci s'applique également si le paquet d'implémentation a `exports` dans son package.json.
+
+#### `AUTRES_FICHIERS.txt`
```
Comment (sandersn) [inline README.fr.md:361]: Leave this as OTHER_FILES.txt

File: README.fr.md [hunk 5]
```diff
@@ -0,0 +1,655 @@
...
+
+Si un fichier n'est ni testé ni référencé dans `index.d.ts`, ajoutez-le à un fichier nommé `AUTRES_FICHIERS.txt`. Ce fichier est une liste d'autres fichiers qui doivent être inclus dans le paquetage typings, un fichier par ligne.
```
Comment (sandersn) [inline README.fr.md:363]: and here

File: README.md [hunk 6]
```diff
@@ -2,8 +2,7 @@
...
-and [日本語](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md)!*
+*You can also read this README in [Español](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md), [한국어](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md), [Русский](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md), [简体中文](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.zh-Hans.md), [Português](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md), [Italiano](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md), [日本語](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md) and [French](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.fr.md)!*
```
Comment (sandersn) [inline README.md:5]:
```suggestion
*You can also read this README in [Español](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md), [한국어](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md), [Русский](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md), [简体中文](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.zh-Hans.md), [Português](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md), [Italiano](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md), [日本語](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md) and [Français](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.fr.md)!*
```

**Review by sandersn — CHANGES_REQUESTED** [review]
J'ai ecrit mes commentaires en anglais parce que j'ai oublie la plupart de mon francais.

File: README.fr.md (hunk 2 above)
Comment (Strawbang) [reply README.fr.md:143]: I'm not sure either, which is why I didn't change the word.

**Comment by Strawbang** [comment]
> J'ai ecrit mes commentaires en anglais parce que j'ai oublie la plupart de mon francais.

no worries :)

**Review by sandersn — APPROVED** [review]

**Pull request merged**